specs/IsPinchZoomEnabled.md

```diff
@@ -0,0 +1,71 @@
+# Background
+
+There are two types of zoom in Chromium: Browser Zoom and Pinch-Zoom:
+- Browser zoom is what you get by using Ctrl + +(plus) or – (minus) or Ctrl + Mousewheel. 
+- Pinch-zoom is activated by using a pinch gesture on a touchscreen. 
```

Comment:
For naming this, in the ADO PR you had a different name I think like page scale? It may be good to distinguish between the kind of zoom (page scale?) and the end user action that can modify it (pinch zoom) if we have other ways of interacting with this kind of zoom, like adding APIs to modify it in the future. 

In that case if we name this API IsPinchZoomEnabled and the underlying kind of zoom is page scale I expect that IsPinchZoomEnabled can disable the end user action but doesn't apply to the underlying page scale and that page scale APIs will still work.

If that's all correct, then I would call this zoom page scale in the documentation but leave the API name as is.

Reply:
I updated the documentation, also added detail about this api doesn't modify the underlying page scale factor in Remarks